README.md

```diff
@@ -2,8 +2,10 @@
 
 Package gads provides a wrapper for the Google Adwords SOAP API.  Based off of
 [colinmutter/gads](https://github.com/colinmutter/gads), this version
-was updated to support v201710 and additional functionality that was missing from 
-the current versions.  
+was updated to support v201710 and additional functionality that was missing from
+the current versions.
+
+**Note:** This repository should be kept public  since we are held to the original license [here](https://github.com/Getsidecar/gads/blob/master/googleads/LICENSE).
```

Comment:
nit, but I try to avoid "click here" style links since they have relatively poor UX ([This article goes into some detail about the disadvantages](https://medium.com/@heyoka/dont-use-click-here-f32f445d1021)). There may be a better way than what I do, but I like to use the phrase describing the thing being linked as the link, like:
```suggestion
**Note:** This repository should be kept public since we are held to [the original license](https://github.com/Getsidecar/gads/blob/master/googleads/LICENSE).
```